specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json

```diff
@@ -0,0 +1,4002 @@
+          "readOnly": true
+        }
+      },
+      "required": [
+        "name"
+      ]
+    },
+    "StoragePolicyList": {
+      "description": "A list of available storage policies",
+      "properties": {
+        "value": {
+          "description": "The total storage policies",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/StoragePolicy"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "HcxEnterpriseSiteList": {
+      "description": "A paged list of HCX Enterprise Sites",
+      "properties": {
+        "value": {
+          "description": "The items on a page",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/HcxEnterpriseSite"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "HcxEnterpriseSite": {
+      "description": "An HCX Enterprise Site resource",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "The properties of an HCX Enterprise Site resource",
+          "$ref": "#/definitions/HcxEnterpriseSiteProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "HcxEnterpriseSiteProperties": {
+      "description": "The properties of an HCX Enterprise Site",
+      "properties": {
+        "activationKey": {
+          "description": "The activation key",
+          "type": "string",
+          "readOnly": true
+        },
+        "status": {
+          "description": "The status of the HCX Enterprise Site",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Available",
+            "Consumed",
+            "Deactivated",
+            "Deleted"
+          ],
+          "x-ms-enum": {
+            "name": "HcxEnterpriseSiteStatus",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "Arc": {
+      "description": "An Arc configuration resource",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "The properties of an Arc resource",
+          "$ref": "#/definitions/ArcProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ArcProperties": {
+      "description": "The properties of an Arc resource",
+      "properties": {
+        "erAuthTokenName": {
+          "description": "Name of the Express route authentication token",
+          "type": "string"
+        },
+        "arcResourceGroupName": {
```

Comment:
what RBAC checks are performed (and how) on the specified subscription + RG here?  How do you know the user has permissions to deploy ARC fabric resources here?

Reply:
We are going to be removing ARC related elements from this version of the API.